word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from bs4 import BeautifulSoup
+
+def wordFindSite ():
+    wordFind_dict = {}
+    for word_length in range(3,12):
+        try:
+            req = requests.get("http://www.wordfind.com/{0}-letter-words/".format(word_length))
+        except:
+            pass
+        
+        unextracted = BeautifulSoup(req.text, 'html.parser').find_all('li')
+        
+        def extract (htmlList):
+            words = []
+            
+            finding_words = list(filter(lambda x: True if x.get('class') == ['defLink'] else False, htmlList))
```

Comment:
Do `lambda x: x.get('class') == ['defLink']`
because the conditional statement is redundant